README.md

```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
 
 `make mkdocs`: Run MkDoc's docker image, building and hosting the html files on `localhost:8000`  
 
+# Local run
+## Prepare:
+
+1. Build executable:
+    ```
+    make build
+    ```
+2. Import keys from Devnet:
+    ```
+    build/chainbridge accounts import --privateKey "improve nominee response kangaroo keen gain antenna pepper spike credit pony parrot" --sr25519
+    ```
+    ```
+    build/chainbridge accounts import --privateKey 5502ab637600e552b45d0608f8f2888a5feafa2a5df1bc67eb9ecc69a9c6b990
+    ```
+    ```
+    build/chainbridge accounts import --privateKey a5485b158e045febc939d9cedc79910f26efe48c31f65643fba3e6dd0f734282
+    ```
+    ```
+    Enter a password after a prompt appears: <any_password>
+    ```
+3. Copy `config/config.json.example` to `config/config.json` folder:
+    ```
+    cp config/config.json.example config/config.json
+    ```
+    
+## Run app in docker:
+
+    docker-compose up
```

Comment:
Please, wrap the command with code block

Reply:
I tried before and I produces extra symbols on top. I think this is how README syntax works.
![image](https://user-images.githubusercontent.com/10967748/173039234-e9c34c2f-279a-46f0-9e54-bcea5cac8863.png)

Reply:
fixed